Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.2.1.plan

```diff
@@ -1,11 +1,12 @@
 {
     "timeout": 1800,
     "count": 1,
-    "local_git_archive": "PerformanceTests/MotionMark@86daf4fcece9d2dafe67ef0de1e23bda72b56378",
-    "github_source": "https://github.com/WebKit/WebKit/tree/86daf4fcece9d2dafe67ef0de1e23bda72b56378/PerformanceTests/MotionMark",
+    "local_git_archive": "PerformanceTests/MotionMark@4b28b98d37306eb5ae9ab8bca504590295f64210",
```

Comment:
where is this revision coming from?

Reply:
This revision is the most recent update to the MotionMark harness, from @grorg in September 2022.
https://github.com/WebKit/WebKit/commit/4b28b98d37306eb5ae9ab8bca504590295f64210

Reply:
Per discussion with smfr: we actually want to leave this revision as-is. Will update to reflect this shortly.